ci-operator/step-registry/netobserv/perf-test/deploy/netobserv-perf-test-deploy-ref.yaml

```diff
@@ -34,6 +34,9 @@ ref:
     - name: PATCH_FLOWLOGS_IMAGE
       default: "false"
       documentation: Patch NOO CSV with custom FLP image
+    - name: MULTISTAGE_PARAM_OVERRIDE_SAMPLING
+      default: ""
+      documentation: Set to override sampling rate
   dependencies:
```

Comment:
@Amoghrd I just realized, could we also add OVERRIDE for [LokiEnable param](https://github.com/openshift-eng/ocp-qe-perfscale-ci/blob/netobserv-perf-tests/scripts/netobserv/flows_v1beta2_flowcollector.yaml#L47)